app/portainer/components/DetailsTable/DetailsTableKeyValueRow.tsx

```diff
@@ -0,0 +1,15 @@
+import { ReactNode } from 'react';
+
+interface Props {
+  children?: ReactNode;
+  keyProp: string;
```

Comment:
what about `label`?
prop is a little redundant.

Reply:
Nice alternative - I'll change it to that - 'key' was reserved for repeating react components